src/main/java/com/healthmarketscience/jackcess/impl/RowImpl.java

```diff
@@ -101,7 +101,6 @@ public Double getDouble(String name) {
   }
 
   @Override
-  @SuppressWarnings("deprecation")
```

Comment:
So i tried to apply these changes last night and i could no longer compile the project with these changes.  I'm not sure what version of java you are using, but i believe all of these deprecation suppressions are still needed (the project is compiled with java 8).